source/uwp/Renderer/lib/pch.h

```diff
@@ -57,3 +57,9 @@
 #include <windows.foundation.collections.h>
 #include <windows.ui.xaml.shapes.h>
 #include <windows.ui.xaml.markup.h>
+
+#include <winrt/base.h>
+#include <winrt/Windows.UI.Xaml.Automation.h>
+#include <winrt/Windows.UI.Xaml.Automation.Peers.h>
+#include <winrt/Windows.UI.Xaml.Controls.h>
```

Comment:
nice! Thanks for getting this done and getting us started on the winrt/cpp route !